src/Export/Skills/other.txt

```diff
@@ -395,12 +395,12 @@ local skills, mod, flag, skill = ...
 	fromItem = true,
 	statMap = {
 		["support_greater_spell_echo_area_of_effect_+%_per_repeat"] = {
-			mod("AreaOfEffect", "INC", nil),
+			mod("AreaOfEffect", "INC", nil, 0, 0, { type = "Condition", var = "CastOnFrostbolt" }),
```

Comment:
Is this meant to be neg = true like the .lua?

Reply:
yes, its meant to be indeed. it's fixed now